src/ui/components/atoms/label-icon/styles.scss

```diff
@@ -0,0 +1,47 @@
+@import '../../../../globals/variables';
+@import '../../../../globals/mixins';
+
+.container {
```

Comment:
You have to create only the necessary space for small elements, only icons, think that these items will be used to implement new molecules, this approach does not work